docs/index.md

```diff
@@ -12,7 +12,7 @@ Tripper
 Getting started
 ---------------
 * [Tutorial]
-* [Discovery of custom backends]
+* [Documentation]
```

Comment:
This is unclear, needs a better title.

Reply:
As discussed, this comes from the main readme file.